test/JDBC/input/pivot-vu-verify.sql

```diff
@@ -350,6 +336,11 @@ PIVOT (
 ) AS pvt
 GO
 
+
+-- Test view of a stmt with pivot operator
+SELECT * FROM pivot_view
```

Comment:
We generally avoid using select * and also use order by some id to make sure resuts of the tests are consistent always